packages/grafana-data/src/dataframe/processDataFrame.ts

```diff
@@ -434,11 +434,22 @@ export function sortDataFrame(data: DataFrame, sortIndex?: number, reverse = fal
       return {
         ...f,
         values: f.values.map((v, i) => f.values[index[i]]),
+        nanos: f.nanos?.map((n, i, nanos) => nanos[index[i]]),
```

Comment:
this will add a `nanos: undefined` attribute to every field without nanos. if that is a problem, i can adjust the code to only assign the `nanos` attribute when it exists, but that will make the code longer. WDYT?